### PR TITLE
Moving folders in correspondance with #41

### PR DESCRIPTION
**Pull request by haixuanTao**
Simply moving crate to mirror our discussion without touching at code our names

### CODE REVIEW

File: libraries/core/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,6 @@ license = "Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```
Comment (phil-opp) [inline libraries/core/Cargo.toml:7]: We should probably rename the package from `dora-common` to `dora-core` too?

**Review by phil-opp — APPROVED** [review]
Looks good to me!

We might want to rename some crates to adjust them to their folder paths. Also I'm not sure if I like the name "serialisation" instead of "message" (if we choose serialisation, we should probably use the American English spelling "serialization" instead).

File: libraries/core/Cargo.toml (hunk 1 above)
Comment (haixuanTao) [reply libraries/core/Cargo.toml:7]: Yep done.

**Comment by haixuanTao** [comment]
Do you have any other specific folder in mind? We can choose to prefix some of the crate names with `api` or `extensions` to reflect the folder structure, is that what you were thinking?

**Comment by phil-opp** [comment]
I think I'm happy with all crates except the `dora-message` crate under `libraries/extensions/serialisation`. I think we should either name the folder `libraries/extensions/message` or rename the crate to `dora-serialization`.

**Comment by haixuanTao** [comment]
Ok. I chose the name `message` as it will include things related to the message design and not just serialization. I also fear that `serialization` might be reductive as it is a verb on something very specific. I will move it to `message` Thanks for your feedback.

**Review by phil-opp — APPROVED** [review]

**Pull request merged**